w5e4.py

```diff
@@ -0,0 +1,102 @@
+# "w5e4"
+
+# connected vessels
+'''
+ Write a function called interleave that gets one or more iterable parameters, and returns a list of intertwined members.
+'''
+
+# function to interleave lists
+def interleave(*iterable):
+    longest = max(len(i) for i in iterable)
+    result = []
+    j = 0
+    #Run like this on the loop because it is important that the first members of each list appear first
+    while j < longest:
+        for i in range(len(iterable)):
```

Comment:
try to reduce these for loops using one of these tools - map\filter\reduce\listcomprehention\itertools.chain\zip